### ISSUE TITLE
Support?

### ISSUE THREAD

**Issue by techker** [Fleker/CumulusTV#303]
Hey guy's is this project not being worked on anymore?not supported? cause it seems pretty dead to me?no more posts in Reddit and here seems very quite?

**Comment by nextux** [comment]
I am to understand that this was a hobbiest project and @Fleker is now working for Google full-time.  I imagine he works on this in his spare time which is probably sparce when starting a new job.

**Comment by Fleker** [comment]
Yes, the project is currently in hiatus. I've thought about working on it again, although I'd first need to rewrite the app from scratch since it's not maintainable in its current form.

**Comment by maddy-mad** [comment]
Hey, @Fleker any plans for updating this app or any similar project in the pipeline?
